odoo-11.0/addons_custom/payslip/data/button_display_payslip.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_employee_form_inherit" model="ir.ui.view">
+        <field name="name">hr.employee.form.inherit</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='toggle_active']" position="after">
+                <button name="action_view_payslips" type="object"
+                        class="oe_stat_button" string="View Payslip"
+                        icon="fa-file-text-o"
+                        attrs="{'invisible': [('id', '=', False)]}"
```

Comment:
attrs="{'invisible': [('id', '=', False)]}" có ý nghĩa là gì

Reply:
attrs được sử dụng để ẩn hiện button, nếu id là failse thì sẽ ẩn button này đi trong th của bài toán thì trường này không cần thiết nhưng có thêm vào để check cũng không sai ạ